cmd/operator-sdk/run/local.go

```diff
@@ -51,6 +52,9 @@ type runLocalArgs struct {
 
 func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
 	prefix := "[local only] "
+
+	fs.StringVar(&c.watchNamespace, "watch-namespace", "",
+		prefix+"The namespace where the operator watches for changes.")
```

Comment:
```suggestion
		prefix+"The namespace where the operator watches for changes. Note that the watch-namespace can be empty or a List of namespaces such as "nsa,nsb" for cluster scoped projects. However, the default matrics implementation requires that it contains the namespace where the operator is `deployed` in. ")
```

he

Reply:
Feel free to re-phase ^

Reply:
HI @nikhil-thomas. 

I'd like to clarify in the description. Note that now(master) the default implementation scaffolded is handling multi-namespaces. See:  https://github.com/camilamacedo86/operator-sdk/blob/master/internal/scaffold/cmd.go#L142